providers/shared/layers/Segment/id.ftl

```diff
@@ -87,7 +87,9 @@
                     "Children" : [
                         {
                             "Names" : "Enabled",
-                            "Types"  : BOOLEAN_TYPE
+                            "Description" : "Don't use - here to override automatically added enabled attribute",
+                            "Types"  : BOOLEAN_TYPE,
+                            "Default" : false
                         },
```

Comment:
Might be better to say Deprecated instead of Don't use

Reply:
done